src/components/Capture/withCrossDeviceWhenNoCamera.tsx

```diff
@@ -42,11 +42,12 @@ const withCrossDeviceWhenNoCamera = <P extends CaptureComponentProps>(
         hasCamera,
         requestedVariant,
         step,
-      } = this.props
-
+        photoCaptureFallback,
+      } = { ...this.props, photoCaptureFallback: true }
```

Comment:
```suggestion
        photoCaptureFallback = true,
      } = this.props
```

Reply:
Thank you for the review! I'll apply the changes and update the section in the README, should be an official change :)